vtex/faststore
# Feat: Adds `CartSidebar` and `OrderSummary`

## What's the purpose of this pull request?

- Adds `CartSidebar` and `OrderSummary`

- Review CartSidebar structure:
  - Creates `CartSidebar`, `CartSidebarList` and `CartSidebarFooter`
  
- Moves `OrderSummary` to faststore/components
	- change data-tested
	- review structure
	- adds styles
	- changes imports
	- verify on CartSidebar
	- Creates component doc page

## How to test it?

<img width="300" alt="cart_side_bar" src="https://user-images.githubusercontent.com/3356699/229945861-3907fe67-8a73-47d7-844d-62dae705f092.png">

You can test the [OrderSummary](https://faststore-site-git-feature-cart-sidebar-fs-774-faststore.vercel.app/components/molecules/order-summary) and [CartSidebar](https://faststore-site-git-feature-cart-sidebar-fs-774-faststore.vercel.app/components/organisms/cart-sidebar) through the docs preview.
Or
You can test the usage of this component in the `faststore/core` application:

- Run yarn dev
- Adds some items to cart
- Open the Cart and check if the component works the same way as [before](https://starter.vtex.app).

### Starters Deploy Preview

https://github.com/vtex-sites/starter.store/pull/42
